nixos/modules/module-list.nix

```diff
@@ -1788,6 +1788,7 @@
   ./system/activation/activatable-system.nix
   ./system/activation/activation-script.nix
   ./system/activation/bootspec.nix
+  ./system/activation/nixos-init.nix
```

Comment:
Release notes entry would be great

Reply:
Done now. Please let me know if you would want some other information in the release note.